### DISCUSSION TITLE
splitting mesh problems

### DISCUSSION THREAD

**Discussion by mcacace** [idaholab/moose#26258] [Q&A Meshing] [answered]
Dear all, I am trying to split a mesh (from gmsh), but despite several efforts (trying it on several machine) it doesn't work, that is, the split-mesh option is not read in and moose starts the input file as in a normal run (even when I have only the mesh block). Not sure what's wrong, since on another mesh it works just fine. Maybe someone can help, thanks.
I do not upload here the original mesh (since it is pretty heavy), but in case I can share on demand. 
mauro

**Comment by mcacace** [comment, marked as answer]
I think I found the culprit ... Apologies for posting!

**Reply by GiudGiud** [reply]
no worries. Thanks for letting us know.
